docs/python/design.md

```diff
@@ -548,55 +547,55 @@ class MyBadEnum(str, Enum):
+{% include requirement/SHOULD id="python-errors-azure-exceptions" %} [`azure-core` 패키지에서 기존 예외 타입](https://azuresdkdocs.blob.core.windows.net/$web/python/azure-core/1.9.0/index.html#azure-core-library-exceptions)을 별도로 생성하는 대신, 기존 예외 타입을 사용하는 것을 선호합니다.
 
-{% include requirement/MUSTNOT id="python-errors-use-standard-exceptions" %} create new exception types when a [built-in exception type](https://docs.python.org/3/library/exceptions.html) will suffice.
+{% include requirement/MUSTNOT id="python-errors-use-standard-exceptions" %} [기본 제공 예외 타입](https://docs.python.org/3/library/exceptions.html)으로 충분한 경우라면 새로운 예외 타입을 생성하지 마세요.
 
-{% include requirement/SHOULDNOT id="python-errors-new-exceptions" %} create a new exception type unless the developer can handle the error programmatically.  Specialized exception types related to service operation failures should be based on existing exception types from the [`azure-core`](https://azuresdkdocs.blob.core.windows.net/$web/python/azure-core/1.9.0/index.html#azure-core-library-exceptions) package.
+{% include requirement/SHOULDNOT id="python-errors-new-exceptions" %} 개발자가 프로그램적으로 오류를 처리할 수 있는 경우가 아니라면, 새로운 예외 타입을 생성하지 마세요. 서비스 작업 실패와 관련된 특수화된 예외 타입은 [`azure-core`](https://azuresdkdocs.blob.core.windows.net/$web/python/azure-core/1.9.0/index.html#azure-core-library-exceptions) 패키지의 기존 예외 타입을 기반으로 해야 합니다.
 
-For higher-level methods that use multiple HTTP requests, either the last exception or an aggregate exception of all failures should be produced.
+여러 개의 HTTP 요청을 사용하는 상위 수준 메서드의 경우, 마지막 예외 또는 모든 실패의 집합적인 예외(aggregated exception)를 생성해야 합니다.
 
-{% include requirement/MUST id="python-errors-rich-info" %} include any service-specific error information in the exception.  Service-specific error information must be available in service-specific properties or fields.
+{% include requirement/MUST id="python-errors-rich-info" %} 예외에는 서비스별 오류 정보를 포함해야 합니다. 서비스별 오류 정보는 서비스별 속성이나 필드에 제공되어야 합니다.
 
-{% include requirement/MUST id="python-errors-documentation" %} document the errors that are produced by each method. Don't document commonly thrown errors that wouldn't normally be documented in Python (e.g. `ValueError`, `TypeError`, `RuntimeError` etc.)
+{% include requirement/MUST id="python-errors-documentation" %} 각 메서드에서 생성되는 오류를 문서화해야 합니다. 일반적으로 Python에서 문서화되지 않는 일반적으로 발생하는 오류 (예: `ValueError`, `TypeError`, `RuntimeError` 등)를 문서화하지 마세요.
 
-### Authentication
+### 인증
 
 {% include requirement/MUST id="python-auth-credential-azure-core" %} use the credentials classes in `azure-core` whenever possible.
 
-{% include requirement/MUST id="python-auth-policy-azure-core" %} use authentication policy implementations in `azure-core` whenever possible.
```

Comment:
Translation of original line 569 seems missing - the next translation is for original line 567.